apps/web-client/src/lib/db/cr-sqlite/__tests__/reconciliation-order.test.ts

```diff
@@ -84,22 +83,49 @@ describe("Suppliers order creation", () => {
 		const res2 = await getReconciliationOrder(db, reconOrderId);
 
 		expect(res2).toMatchObject({
-			customer_order_line_ids: null,
 			id: 1,
 			supplier_order_ids: "[1]",
 			finalized: 0
 		});
 
-		await addOrderLinesToReconciliationOrder(db, 1, ["123", "435", "324"]);
+		await addOrderLinesToReconciliationOrder(db, 1, [
```

Comment:
Do we have a case to check that existing order line quantity will be updated on conflict?